modules/auth/schema/inspection.go

```diff
@@ -122,3 +123,39 @@ func inspectionPostgresCheckFieldType(typeName string, fieldDetails *schemaField
 	}
 	return nil
 }
+
+// GetCollectionName returns all the collection name for specified database
+func (s *Schema) GetCollectionName(ctx context.Context, project, dbType string) ([]string, error) {
+	switch utils.DBType(dbType) {
+	case utils.Mongo, utils.MySQL, utils.Postgres:
+		collections, err := s.crud.GetCollections(ctx, project, dbType)
+		if err != nil {
+			return nil, err
+		}
+		col := make([]string, len(collections))
+		for key, value := range collections {
+			col[key] = value.TableName
+		}
+		return col, nil
+	default:
+		return nil, errors.New("collections wrongs database")
+	}
+}
+
+// GetSchemaCollection returns schemas of collection aka tables for specified project & database
+func (s *Schema) GetSchemaCollection(ctx context.Context, project, dbType string) (config.Crud, error) {
+	projectConfig := config.Crud{}
+	collections, err := s.GetCollectionName(ctx, project, dbType)
```

Comment:
Here you need to find the schema of only those collections which are already present in the config. You need yo be a lil more alert while coding